### PR TITLE
Update project display

### PR DESCRIPTION
**Pull request by Thinus01**
### General requirements

- Made sure that there are no linter errors.
- Made sure that I used correct flow Gitflow.
- Made sure that I documented my work in a professional way.

### Project requirements
- Implemented the design from the sneak peek wireframes.
- Used methods that I have created in the Blog App.

### CODE REVIEW

File: app/views/users/index.html.erb [hunk 1]
```diff
@@ -1 +1,16 @@
...
+        <div class="user">
+            <img src=<%= user.photo %> alt="User Photo" class="user-photo">
+            <div class="user-details">
+                <h2 class="user-name"><%= user.name%></h2>
+                <h5 class="user-post-count">Number of posts: <%= user.posts_counter %> </h5>
+            </div>
+
+        </div>
+        </a>
+    <% end %>
```
Comment (TedLivist) [inline app/views/users/index.html.erb:14]: - [ ] Kindly add more width to your users on this page based on the wireframe, just as you have done in the other pages.

  image

- [ ] **[OPTIONAL]** It is often a better practice in Rails to use partials for rendering collections in views.

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -1,5 +1,11 @@
...
-  def index; end
+  def index
+    @users = User.all
+  end
 
-  def show; end
+  def show
+    @user = User.find(params[:id])
+    @user_posts = Post.where(author_id: @user.id)
+    @recent_posts = @user.get_recent_posts
```
Comment (TedLivist) [inline app/controllers/users_controller.rb]: - [ ] You have this `recent_posts` here. But in the user show view page you are still calling `user.get_recent_posts` again. Let's try to eliminate unnecessary queries to save time and clean up our code. Kindly remove this line, or use the `@recent_posts` variable in the view instead.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @Thinus01,

Good job so far!
✔️ UI matches the wireframes
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the **[OPTIONAL]** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by uwadonat — APPROVED** [review]
## Hi @Thinus01,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 
### Highlights
- Detailed pull request message ✔️
- Professional readme file ✔️
- No linter errors ✔️

# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**